contracts/sol6/Dao/KyberDAO.sol

```diff
@@ -57,9 +57,9 @@ contract KyberDAO is IKyberDAO, EpochUtils, ReentrancyGuard, Utils5, DaoOperator
 
     // minimum duration in seconds for a campaign
     uint256 public minCampaignDurationInSeconds = 345600; // around 4 days
-    IERC20 public kncToken;
-    KyberStaking public staking;
-    IKyberFeeHandler public feeHandler;
+    IERC20 public immutable kncToken;
+    IKyberStaking public staking;
```

Comment:
After PR deploying staking at Dao constructor, staking could be marked immutable

Reply:
right
will update

Reply:
One other reason why it wasn't set to immutable is because the mock contract allows it to change, but on closer inspection, this can (and has) been fixed.

Same for the dao contract. See latest commit.